Autogen / CANfigurator Fails With Missing Readonly Module

# Bug Report

## Problem and Scope
`CANfigurator` sometimes fails (anywhere from 1 in 4 to 2 in 3) due to a missing module in GitHub Action runners, behavior is intermittent though

A literal rerun can get different results due to cache differences

## Steps to Reproduce
1. Run the `Autogen / CANfigurator` action
2. Repeat 2-3 times

## Testing Environment
GitHub Actions

## Suggested Fix
Add `Readonly` to the <q>Setup Perl</q> step installed modules
```yaml
- name: Setup Perl
  uses: shogo82148/actions-setup-perl
  with:
    install-modules: 'YAML::XS Readonly'
```

## Bug Discovery
```
make: Entering directory '/home/runner/work/Firmware/Firmware/Autogen/CAN'
perl Src/STRUCTparser.pl Doc/GRCAN.CANdo Inc/GRCAN_MSG_DATA.h
Can't locate Readonly.pm in @INC (you may need to install the Readonly module) (@INC entries checked: /home/runner/work/Firmware/Firmware/local/lib/perl5/5.42.0/x86_64-linux /home/runner/work/Firmware/Firmware/local/lib/perl5/5.42.0 /home/runner/work/Firmware/Firmware/local/lib/perl5/x86_64-linux /home/runner/work/Firmware/Firmware/local/lib/perl5 /home/runner/work/Firmware/Firmware/local/lib/perl5/x86_64-linux /home/runner/work/_actions/shogo82148/actions-setup-perl/b6cfc127f1b0a63ac98797800f641494d80f42f8/scripts/lib /home/runner/work/_actions/shogo82148/actions-setup-perl/b6cfc127f1b0a63ac98797800f641494d80f42f8/scripts/lib/perl5 /opt/hostedtoolcache/perl/5.42.0/x64/lib/site_perl/5.42.0/x86_64-linux /opt/hostedtoolcache/perl/5.42.0/x64/lib/site_perl/5.42.0 /opt/hostedtoolcache/perl/5.42.0/x64/lib/5.42.0/x86_64-linux /opt/hostedtoolcache/perl/5.42.0/x64/lib/5.42.0) at Src/STRUCTparser.pl line 8.
BEGIN failed--compilation aborted at Src/STRUCTparser.pl line 8.
make: *** [Makefile:34: STRUCTparser] Error 2
make: Leaving directory '/home/runner/work/Firmware/Firmware/Autogen/CAN'
```
## Estimated Severity
Critical, a fix is needed soon but the problem is currently circumvented when encountered

--- a/.github/workflows/Autogen.yml
+++ b/.github/workflows/Autogen.yml
@@ -26,8 +26,8 @@ jobs:
         uses: shogo82148/actions-setup-perl@b6cfc127f1b0a63ac98797800f641494d80f42f8  # v1.37.1
         with:
           install-modules: |
-            'Readonly'
-            'YAML::XS'
+            Readonly
+            YAML::XS
 
       - name: Run CAN Parser
         run: |

